.github/dependabot.yml

```diff
@@ -3,20 +3,6 @@
 
 version: 2
```

Comment:
Do we need to keep this file around if the workflow is gone and renovate is handling deps for us?

Reply:
@kevinbarabash Due to the limitation of GitHub Apps, Renovate cannot upgrade GitHub Actions workflow dependencies.